renormalizer/cv/zerot.py

```diff
@@ -0,0 +1,345 @@
+# -*- coding: utf-8 -*-
+# Author: Tong Jiang <tongjiang1000@gmail.com>
+# zero temperature absorption/emission spectrum based on Correction vector - DMRG
+
+
+from renormalizer.mps import Mpo, Mps, solver
+from renormalizer.mps.solver import construct_mps_mpo_2, optimize_mps
+from renormalizer.mps.matrix import (
+    Matrix,
+    tensordot,
+    multi_tensor_contract,
+    ones,
+    einsum
+)
+import numpy as np
+import scipy
+import copy
+
+class SpectraZtCV(object):
+    def __init__(
+        self,
+        mol_list,
+        spectratype,
+        freq_reg,
+        m_max,
+        eta,
+        method="1site",
+        procedure_gs=None,
+        procedure_cv=None
+    ):
+        self.freq_reg = freq_reg
+        self.m_max = m_max
+        self.eta = eta
+        self.mol_list = mol_list
+        assert spectratype in ["abs", "emi"]
+        self.spectratype = spectratype
+        if procedure_gs is None:
+            procedure_gs = [[10, 0.4], [20, 0.2], [30, 0.1], [40, 0], [40, 0]]
+        if procedure_cv is None:
+            procedure_cv = [0.4, 0.4, 0.2, 0.2, 0.1, 0.1] + [0] * 15
+        self.procedure_gs = procedure_gs
+        self.procedure_cv = procedure_cv
+        self.method = method
+
+    def init_mps(self):
+        if self.spectratype == "abs":
+            self.nexciton = 0
+            dipoletype = r"a^\dagger"
+        else:
+            self.nexciton = 1
+            dipoletype = "a"
+
+        dipole_mpo = \
+            Mpo.onsite(
+                self.mol_list, dipoletype, dipole=True
+            )
+        mps, self.mpo = \
+            construct_mps_mpo_2(
+                self.mol_list, self.procedure_gs[0][0], self.nexciton
+            )
+        mps.optimize_config.procdure = self.procedure_gs
+        mps.optimize_config.method = "2site"
+        self.lowest_e = optimize_mps(mps, self.mpo)
+        ket_mps = dipole_mpo.apply(mps, canonicalise=True)
+        self.b_oper = ket_mps.scale(-self.eta)
+
+    def cv_solve(self):
+        self.init_mps()
+        self.cv_mps = Mps.random(
+            self.mol_list, 1-self.nexciton, self.m_max, percent=1.0)
+
+        total_num = 1
+        spectra = []
+
+        # Here did not implement cpu-parallelization
+        for omega in self.freq_reg:
+            print('omega', omega)
```

Comment:
If this is for debugging, I'd suggest to comment this out. If this is for logging, using the `logging` module would be more consistent with other parts of the project.